opencog/cython/opencog/link_value.pyx

```diff
@@ -31,8 +31,6 @@ cdef class LinkValue(Value):
             if is_a(deref(value).get_type(), types.Value):
                 list.append(create_python_value_from_c_value(value))
             else:
-                # TODO: Support Atoms as members of LinkValue requires inheriting
```

Comment:
I believe whole `if` condition could be removed and replaced by ```list.append(create_python_value_from_c_value(value))```

Reply:
Yeah, I guess `create_python_value_from_c_value` will raise any Exceptions.